src/test/java/com/esri/core/geometry/TestOGC.java

```diff
@@ -955,5 +955,86 @@ public void testEmptyBoundary() throws Exception {
 		}
 	}
 	
+	@Test
+	public void testUnionPointWithEmptyLineString() {
+		assertUnion("POINT (1 2)", "LINESTRING EMPTY", "GEOMETRYCOLLECTION (POINT (1 2))");
```

Comment:
I'd expect the union of `POINT (1, 2)` and `LINESTRING EMPTY` to be `POINT (1, 2)`, but this test asserts the result to be `GEOMETRYCOLLECTION (POINT (1 2))`.

Reply:
@mbasmanova Yes, a simpler representation would be better.